presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryStatistics.java

```diff
@@ -63,6 +71,10 @@ public QueryStatistics(
         this.peakMemoryBytes = requireNonNull(peakMemoryBytes, "peakMemoryBytes is null");
         this.totalBytes = requireNonNull(totalBytes, "totalBytes is null");
```

Comment:
Can you add another commit that removes these silly requireNonNulls? Primitives cannot be null.